### PR TITLE
Docs/dashboard documentation

### PR DESCRIPTION
**Pull request by antho1404**
### Project organization

- Closes 
- Related 
- Dependant on https://github.com/liteflow-labs/liteflow-js/pull/155

### Checklist

- [ ] Update related changelogs 
- [ ] Update docs if necessary

### CODE REVIEW

File: docs/pages/dashboard/_meta.json [hunk 1]
```diff
@@ -1,4 +1,7 @@
 {
   "index": "Introduction",
```
Comment (NicolasMahe) [inline docs/pages/dashboard/_meta.json:2]: Let's not forget to add a section "dashboard" in the introduction page

Comment (EmmanuelDrouin) [reply docs/pages/dashboard/_meta.json:2]: Agree, currently missing ;)

**Comment by antho1404** [comment]
Comment from @EmmanuelDrouin in https://github.com/liteflow-labs/liteflow-js/pull/155

----

@antho1404 I'm good with the new orga. One link is not working tho "Configure plugins".

Depending on where you see those "how to" going, I would recommend to maybe group those into topics like "collection", "payment method", etc. and have inside those group the related pages.

I'm saying that as the new pages you added are "import a collection", "add a currency", etc. which are starting points but ideally we would have for example:

- Collection
  - Import a collection
  - Edit a collection
  - Remove a collection

If you follow where I'm going ;)

**Comment by antho1404** [comment]
I do agree with your comment @EmmanuelDrouin. Ideally, that should be the structure, I wanted to have something more "temporary" where we have kind of like "recipes" that could also be used as use cases with a few benefits of why this feature is good and, in your example, edit/remove a collection is kind of secondary

**Comment by antho1404** [comment]
Documentation updated with new structure

**Pull request closed without merging**